src/reducers/entities/emojis.js

```diff
@@ -81,6 +81,20 @@ function nonExistentEmoji(state = new Set(), action) {
         }
         return state;
     }
+    case EmojiTypes.LOAD_NONEXISTENT_EMOJIS: {
```

Comment:
Can we match the other type's name, so update to:
```suggestion
    case EmojiTypes. CUSTOM_EMOJIS_DO_NOT_EXIST: {
```

Reply:
Sure, now updated, thanks